### PR TITLE
Get page/qs from NPM, browserify them with cosmos:browserify

### PR DESCRIPTION
**Pull request by elidoran**
Allows using package manager to update the modules instead of manually placing them in the `client/vendor` folder. It browserify's the files to avoid having to do that manually.

See: [cosmos:browserify](http://github.com/elidoran/cosmos-browserify)

### CODE REVIEW

File: browserify.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+window.page = require('page');
```
Comment (arunoda) [inline browserify.js]: Why do we need to attach this to window?
Why not qs?

**Comment by arunoda** [comment]
Yes. This is super awesome.
You should write a blog post about this `cosmos:browserify`.
It's super useful :)

**Comment by arunoda** [comment]
I'll take this one and merge it.

**Comment by elidoran** [comment]
Your code gets `page` by `window.page`, see [here](https://github.com/meteorhacks/flow-router/blob/master/client/router.js#L338). I thought I should mirror that and assign the value explicitly as `window.page`. Also, I wanted to show your code doesn't have to change to use this new way.

And, you're right. It doesn't need to be assigned like that and router.js can change how it gets the page variable as well.

**Comment by arunoda** [comment]
Yes it is. We it because page.js assign it window by default. I hope we don't need to do it in the `browserify.js`

**Comment by arunoda** [comment]
Here's the diff and it works:

~~~patch
diff --git a/browserify.js b/browserify.js
index 55e901a..7b734f4 100644
--- a/browserify.js
+++ b/browserify.js
@@ -1,2 +1,2 @@
-window.page = require('page');
-qs          = require('qs');
+page = require('page');
+qs   = require('qs');
diff --git a/client/router.js b/client/router.js
index 9b82d64..312bf5e 100644
--- a/client/router.js
+++ b/client/router.js
@@ -335,7 +335,5 @@ Router.prototype._updateCallbacks = function () {
   });
 };

-Router.prototype._page = window.page;
-delete window.page;
-
-Router.prototype._qs = qs;
+Router.prototype._page = page;
+Router.prototype._qs = qs;
\ No newline at end of file
~~~

**Comment by arunoda** [comment]
Once we figure it out what's happening here: https://github.com/elidoran/cosmos-browserify/issues/4
and after applied the above patch, I'm happy with this.

File: package.js [hunk 2]
```diff
@@ -13,9 +18,9 @@ Package.onUse(function(api) {
   api.use('reactive-dict');
 
   api.use('meteorhacks:fast-render@2.3.2', ['client', 'server'], {weak: true});
+  api.use('cosmos:browserify', 'client');
```
Comment (arunoda) [inline package.js]: Let's the add the version here. 

~~~js
api.use('cosmos:browserify@0.1.3', 'client');
~~~

**Comment by arunoda** [comment]
Don't squash the commit. I like to have history.

**Comment by elidoran** [comment]
Okay, unless you have any other requests I'll leave this PR as it is right now.

**Comment by arunoda** [comment]
I'm going to merge this one.

**Comment by arunoda** [comment]
Merged and released with `1.6.0`

**Pull request merged**